tracer/src/Datadog.Trace/AppSec/Waf/IResult.cs

```diff
@@ -4,16 +4,21 @@
 // </copyright>
 
 using System;
+using System.Collections.Generic;
 
 namespace Datadog.Trace.AppSec.Waf
 {
     internal interface IResult : IDisposable
     {
         ReturnCode ReturnCode { get; }
 
+        public bool Block { get; }
+
+        public bool ShouldBeReported { get; }
+
         string Data { get; }
 
-        string[] Actions { get; }
+        IList<string> Actions { get; }
```

Comment:
nit: I would make this a `List<>` rather than an `IList<>`, as no really benefit for our internal types (and removes need to de-virtualize)